data/lark/loadData.mjs

```diff
@@ -1,109 +1,110 @@
-	const organizations = await fetch(`${urlBase}/api/organizations/`, {
-		headers: { 'Authorization': `Token ${token}` },
-	}).then(r => r.json());
-	
-	let org = organizations.find(o => o.name === "Lark Traditional Arts");
-	if (!org) {
-		console.log('Could not find Lark Traditional Arts, creating');
-		org = await fetch(`${urlBase}/api/organizations/`, {
-			method: 'POST',
-			headers: { 
-				'Content-Type': 'application/json',
-				'Authorization': `Token ${token}`, 
-			},
-			body: JSON.stringify({
-				name: "Lark Traditional Arts"
-			}),
-		}).then(res => res.json());
-	}
-	console.log(org);
-	return org;
+  const organizations = await fetch(`${urlBase}/api/organizations/`, {
+    headers: { 'Authorization': `Token ${token}` },
+  }).then(r => r.json());
+  
+  let org = organizations.find(o => o.name === "Lark Traditional Arts");
+  if (!org) {
+    console.log('Could not find Lark Traditional Arts, creating');
+    org = await fetch(`${urlBase}/api/organizations/`, {
+      method: 'POST',
+      headers: { 
+        'Content-Type': 'application/json',
+        'Authorization': `Token ${token}`, 
+      },
+      body: JSON.stringify({
+        name: "Lark Traditional Arts"
+      }),
+    }).then(res => res.json());
+  }
+
+  return org;
 }
 
 async function getEvent(token, org) {
-	let response = await fetch(`${urlBase}/api/events/`, {
-		headers: { 'Authorization': `Token ${token}` },
-	});
-	
-	const events = await response.json();
-	const existingEvent = events.find(event => event.name === "Lark 2021");
-	
-	const event = {
-		organization: org.id,
-		name: "Lark 2021",
-		confirmation_email_from: 'lark-test@example.com',
-		...modules,
-	};
-	if (existingEvent) {
-		event.id = existingEvent.id;
-	}
-	response = await fetch(`${urlBase}/api/events/${existingEvent ? `${event.id}/` : ''}`, {
-		method: existingEvent ? 'PUT' : 'POST',
-		headers: {
-			'Authorization': `Token ${token}`,
-			'Content-Type': 'application/json',
-		},
-		body: JSON.stringify(event),
-	});
-	const json = await response.json();
-	
-	console.log(json);
+  let response = await fetch(`${urlBase}/api/events/`, {
+    headers: { 'Authorization': `Token ${token}` },
+  });
+  
+  const events = await response.json();
+  const existingEvent = events.find(event => event.name === "Lark 2021");
+  
+  const event = {
+    organization: org.id,
+    name: "Lark 2021",
+    confirmation_email_from: 'lark-test@example.com',
+    ...modules,
+  };
+  if (existingEvent) {
+    event.id = existingEvent.id;
+  }
+  response = await fetch(`${urlBase}/api/events/${existingEvent ? `${event.id}/` : ''}`, {
+    method: existingEvent ? 'PUT' : 'POST',
+    headers: {
+      'Authorization': `Token ${token}`,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(event),
+  });
+  const json = await response.json();
+
+  return json;
 }
 
+export default main;
 
-
-
-
+if (import.meta.url === `file://${process.argv[1]}`) {
```

Comment:
woah the `if name == "__main__"` in JS! I've never seen this pattern used like this. Generally i just see it in a separate file. But this works.

Reply:
Yeah, since this is a singular file that acts as a script, this is basically it.  The only reason I did this is that this way it can either be imported as a module or ran as a stand along script.  At this point it's probably only going to be a stand alone script though, so I could remove it if you want.

Reply:
yeah, i just realize how this is super common in python but so rare otherwise.